src/backend/distributed/transaction/coordinated_transaction_management.c

```diff
@@ -20,9 +20,10 @@
 #include "access/twophase.h"
 #include "access/xact.h"
 #include "distributed/connection_management.h"
+#include "distributed/coordinated_transaction_management.h"
+#include "distributed/distributed_transaction_management.h"
```

Comment:
I think the split isn't actually gaining us clarity, and coordinated and distributed transactions are pretty much the same thing, and what currently resides in distributed_transaction_management.h isn't really what the name says. So I don't think the split makes much sense as is. I think I'd be ok with introducing new 'distributed_transaction_identifier' files or such.